File: Upcoder/Other/Liệt kê số chính phương/H_LietKeSCP.py
import math

def is_perfect_square(n):
    temp = int(math.sqrt(n))
    if temp * temp == n:
        return True
    return False

def main():
    n, m = map(int, input().split())
    for i in range(n, m + 1):
        if is_perfect_square(i):
            print(i)

if __name__ == "__main__":
    main()

File: Upcoder/Other/Liệt kê số chính phương/H_LietKeSCP.cs
using System;
namespace SoCP
{
    public class Program
    {
        static void Main(string[]args)
        {
            string []input = Console.ReadLine().Split(' ');
            int n = int.Parse(input[0]);
            int m = int.Parse(input[1]);
            for(int i = n; i <= m; i++)
            {
                if(SCP(i))
                {
                    Console.WriteLine(i);
                }
            }
        }
        static bool SCP(int n)
        {
            double temp = Math.Sqrt(n);
            temp = (int)temp;
            if(temp * temp == n)
                return true;
            return false;
        }
    }
}